lib/sqs.js

```diff
@@ -117,8 +119,18 @@ class SQSDriver extends Driver {
         return attrAcc;
       }, {});
 
+      // To be backwards compatible only apply these values when passed in the message.
+      const extraParams = {};
+
+      if (message.dedupId) {
+        extraParams.dedupId = message.dedupId;
+      }
+      if (message.groupId) {
+        extraParams.groupId = message.groupId;
+      }
+
```

Comment:
This is how a real message looks like, thus it needs to be `attributes.MessageGroupId` and `attributes.MessageDeduplicationId`:

```json
{
    "Records": [
        {
            "messageId": "11d6ee51-4cc7-4302-9e22-7cd8afdaadf5",
            "receiptHandle": "AQEBBX8nesZEXmkhsmZeyIE8iQAMig7qw...",
            "body": "Test message.",
            "attributes": {
                "ApproximateReceiveCount": "1",
                "SentTimestamp": "1573251510774",
                "SequenceNumber": "18849496460467696128",
                "MessageGroupId": "1",
                "SenderId": "AIDAIO23YVJENQZJOL4VO",
                "MessageDeduplicationId": "1",
                "ApproximateFirstReceiveTimestamp": "1573251510774"
            },
            "messageAttributes": {},
            "md5OfBody": "e4e68fb7bd0e697a0ae8f1bb342846b3",
            "eventSource": "aws:sqs",
            "eventSourceARN": "arn:aws:sqs:us-east-2:123456789012:fifo.fifo",
            "awsRegion": "us-east-2"
        }
    ]
}
```